### PR TITLE
[DO NOT MERGE] : dialog_dk1469x-dk-pro: Fix otp_write_key

### PR DESCRIPTION
**Pull request by nkaje**
otp_write_key reads a base64 encoded key. When encoded, the key
is longer than 32 bytes. struct.unpack expects it to be decoded
and 32 bytes long. Hence decode it and then use.

Signed-off-by: Naveen Kaje <naveen.kaje@juul.com>

### CODE REVIEW

File: hw/bsp/dialog_da1469x-dk-pro/otp_tool.py [hunk 1]
```diff
@@ -179,9 +180,9 @@ def otp_write_key(infile, index, segment, uart):
             else:
                 # read key from base64 encoded AES key file
                 buf = f.read()
-                if isBase64(buf):
-                    if len(buf) != 32:
-                        raise SystemExit("AES key file has incorrect length")
+                buf = base64.b64decode(buf)
```
Comment (nkaje) [inline hw/bsp/dialog_da1469x-dk-pro/otp_tool.py]: Now that isBase64() not used, could we remove it?

Comment (agross-korg) [reply hw/bsp/dialog_da1469x-dk-pro/otp_tool.py]: yes

Comment (agross-korg) [reply hw/bsp/dialog_da1469x-dk-pro/otp_tool.py]: else:
                # read key from base64 encoded AES key file
                buf = f.read()

                try:
                    buf = base64.decodestring(buf)
                except ValueError:
                    raise SystemExit("Invalid base 64 file")

                if len(buf) != 32:
                    raise SystemExit("AES key file has incorrect length")

Comment (agross-korg) [reply hw/bsp/dialog_da1469x-dk-pro/otp_tool.py]: this would catch errors without spewing to stderr

Comment (nkaje) [reply hw/bsp/dialog_da1469x-dk-pro/otp_tool.py]: Done and new PS uploaded.

**Comment by nkaje** [comment]
These changes are being made in https://github.com/apache/mynewt-core/pull/2061. Do not merge 2056

**Pull request closed without merging**